lib/activerecord-import/import.rb

```diff
@@ -592,7 +592,6 @@ def import_helper( *args )
         if models
           import_with_validations( column_names, array_of_attributes, options ) do |validator, failed|
             models.each_with_index do |model, i|
-              model = model.dup if options[:recursive]
```

Comment:
This seems to be the cause of the N+1 queries when models had validations that referenced associations. The associations are not included in the duped object, so they had to be loaded.